src/Faker/Provider/Miscellaneous.php

```diff
@@ -318,6 +318,10 @@ public static function currencyCode()
      */
     public static function emoji()
     {
+        if (!function_exists('json_decode')) {
+            return new \RuntimeException('The JSON extension is required to generate Emojis.');
```

Comment:
```suggestion
            throw new \RuntimeException('The JSON extension is required to generate Emojis.');
```

Can we also add this to the DocBlock, if we insist on throwing an exception here?

Reply:
@krsriq  Since we loosen the BC check rules to allow changing protected properties, it should be now possible to use it without json extension (#124)

Reply:
Yes, we don't need `json_decode` any more.